notify/pushover.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env sh
+
+#Support PUSHOVER.com api
```

Comment:
PUSHOVER.net  ?

Reply:
updated... sorry, comment was from an existing file and I didn't double check the comments.